NEWS.rst

```diff
@@ -16,6 +16,7 @@ Version 1.0.0, 2019-10-04
 New features
 ------------
 
+* #29760: Best practises updated concerning OS2Sync integration
```

Comment:
This goes in the "Under development" section.

Reply:
Fanx! good I am not flying airbus.